libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -961,6 +962,22 @@ const AP_Param::GroupInfo AP_OSD_Screen::var_info[] = {
     // @Description: Vertical position on screen
     // @Range: 0 15
     AP_SUBGROUPINFO(restvolt, "RESTVOLT", 58, AP_OSD_Screen, AP_OSD_Setting),
+
+    // @Param: FENCE_EN
+    // @DisplayName: FENCE_EN
+    // @Description: Displays indication of fence enable and breach
+    // @Values: 0:Disabled,1:Enabled
+
+    // @Param: FENCE_X
+    // @DisplayName: FENCE_X
+    // @Description: Horizontal position on screen
+    // @Range: 0 29
+
+    // @Param: FENCE_Y
+    // @DisplayName: FENCE_Y
+    // @Description: Vertical position on screen
+    // @Range: 0 15
+    AP_SUBGROUPINFO(fence, "FENCE_EN", 59, AP_OSD_Screen, AP_OSD_Setting),
```

Comment:
```suggestion
    AP_SUBGROUPINFO(fence, "FENCE", 59, AP_OSD_Screen, AP_OSD_Setting),
```

I think this was my first wild-assed solution - which was wrong and needs to be reverted.